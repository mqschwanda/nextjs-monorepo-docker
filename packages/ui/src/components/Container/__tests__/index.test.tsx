import { render } from '@testing-library/react';
import { Container } from '..';

describe('<Container />', () => {
  it('renders correctly', () => {
    const testId = 'Container';
    const chilren = 'children...';

    const { getByTestId, asFragment } = render((
      <Container
        testId={testId}
      >
        { chilren }
      </Container>
    ));

    const element = getByTestId(testId);
    expect(element).toBeTruthy();

    expect(asFragment()).toMatchSnapshot();
  });
});
