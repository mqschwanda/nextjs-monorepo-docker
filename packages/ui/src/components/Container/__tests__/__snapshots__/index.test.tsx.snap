// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`@mqs/ui components <Container /> renders 1`] = `
<DocumentFragment>
  <div
    class="MuiContainer-root MuiContainer-maxWidthLg css-1oqqzyl-MuiContainer-root"
    data-testid="Container"
  >
    children...
  </div>
</DocumentFragment>
`;
